runner/ollamarunner/runner.go

```diff
@@ -560,6 +560,9 @@ func (s *Server) forwardBatch(pendingBatch batchState) (nextBatch batchState, er
 		}
 
 		seq.inputs = seq.inputs[len(seq.pendingInputs):]
+		if seq.startedAt.IsZero() {
+			seq.startedAt = time.Now()
+		}
```

Comment:
This will avoid counting the batch formation for this sequence but I believe that it will include batch formation time for later sequences.